tcf_website/views/search.py

```diff
@@ -0,0 +1,106 @@
+"""Views for search results"""
+import os
+import json
+import requests
+
+from django.shortcuts import render
+from django.contrib.auth.decorators import login_required
+
+@login_required
+def search(request):
```

Comment:
Can you add some tests for these functions, perhaps by mocking ElasticSearch?

https://docs.djangoproject.com/en/3.0/intro/tutorial05/#test-a-view